FruitFruit/FruitFruit/Utilities/Storage.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  Storage.swift
+//  FruitFruit
+//
+//  Created by Junyeong Park on 2022/07/19.
+//
+
+import Foundation
+
+struct Storage {
+    static let defaults = UserDefaults.standard
+    
+    func setFruitUser(fruitUser: FruitUser) {
+        let encoder = JSONEncoder()
+        if let encoded = try? encoder.encode(fruitUser) {
+            UserDefaults.standard.setValue(encoded, forKey: "FruitUser")
+        }
```

Comment:
```suggestion
        let encoder = JSONEncoder()
        guard let encoded = try? JSONEncoder().encode(fruitUser)  else { return }

        UserDefaults.standard.setValue(encoded, forKey: "FruitUser")
```

guard let으로하면 빠르게 리턴되고, 인덴트가 줄어서 코드 가독성도 높아집니다 ~